### ISSUE TITLE
SC 3.2.6 Findable Help - Help Mechanism Priority

### ISSUE THREAD

**Issue by omar-bonilla** [w3c/wcag#1434]
On Behalf of Thomson Reuters:

Thomson Reuters appreciates the need for various forms of assistance to users as they complete processes throughout a site, but are the help mechanisms to be prioritized in some order, given multiple mechanisms available on a site? Human contact supports the broadest group, and thus should be first in our opinion.

**Comment by scha14** [comment]
Draft response: 

> The success criterion does not cover the priority of ways of finding help. It only discusses some examples of finding help.

**Comment by alastc** [comment]
On the call the group agreed this response:
<hr>
The success criterion does put the ways of help in a rough prioritisation order, but it does not cover prioritisation when the site provides multiple ways. There are too many variables to provide direction for this aspect, for example, the "Human contact mechanism" (messaging) might be better supported by the organisation than the "Human contact details" (phone). 
The group believes this aspect should be up to the site/organisation.
<hr>
In addition, we are looking at a small understanding document update to include that information.
